spykes/datasets.py

```diff
@@ -29,25 +30,23 @@ def load_reward_data(dpath='~/spykes_data/reward/'):
     sess_four_mat: .mat file
         Monkey M, Session 4
     """
-
+    dpath = os.path.expanduser(dpath)
     if not os.path.exists(dpath):
         os.makedirs(dpath)
 
     fname = os.path.join(dpath, 'Mihili_07112013.mat')
 
     if not os.path.exists(fname):
 
-        urllib.urlretrieve('https://ndownloader.figshare.com/files/5652051',
-                           fname)
+        _fetch_file('https://ndownloader.figshare.com/files/5652051', fname)
```

Comment:
ok here's a package that accomplishes what `_fetch_file` does but with minimal dependencies. Could either add this in as a dependency for spykes, or freeze it and store it as an external inside of spykes. WDYT?

https://github.com/choldgraf/download